integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/container/ContainerSuite.java

```diff
@@ -83,6 +83,7 @@ public void startHiveContainer() {
           HiveContainer.Builder hiveBuilder =
               HiveContainer.builder()
                   .withHostName("gravitino-ci-hive")
+                  .withImage("unknowntpo/gravitino-ci-hive:latest")
```

Comment:
I use my hive image here, we need to upload newly created `gravitino-ci-hive` image before merging.

Reply:
Hi, @unknowntpo , It's not good practice to use your image here, @xunliu Please help to verify if changes about the Hive image are reasonable, If everything is OK, we can release the image before this PR is merged, then @unknowntpo can change the image to the newly released one.

@unknowntpo Another point is that you may need to modify the related document about the Hive image, please see 
https://github.com/datastrato/gravitino/blob/14297cddc894a4d47851ddfbcaec50bc547e0387/docs/docker-image-details.md?plain=1#L87C1-L88C1

Reply:
ok, if @xunliu find this modification reasonable and push new image to docker hub, I'll update the changelog in docs.

Reply:
I've add some todos in the PR description.

Reply:
> Hi, @unknowntpo , It's not good practice to use your image here, @xunliu Please help to verify if changes about the Hive image are reasonable, If everything is OK, we can release the image before this PR is merged, then @unknowntpo can change the image to the newly released one.
> 
> @unknowntpo Another point is that you may need to modify the related document about the Hive image, please see https://github.com/datastrato/gravitino/blob/14297cddc894a4d47851ddfbcaec50bc547e0387/docs/docker-image-details.md?plain=1#L87C1-L88C1

@yuqi1129 Can you help release this Docker image to hub?

Reply:
> > Hi, @unknowntpo , It's not good practice to use your image here, @xunliu Please help to verify if changes about the Hive image are reasonable, If everything is OK, we can release the image before this PR is merged, then @unknowntpo can change the image to the newly released one.
> > @unknowntpo Another point is that you may need to modify the related document about the Hive image, please see [`14297cd`/docs/docker-image-details.md?plain=1#L87C1-L88C1](https://github.com/datastrato/gravitino/blob/14297cddc894a4d47851ddfbcaec50bc547e0387/docs/docker-image-details.md?plain=1#L87C1-L88C1)
> 
> @yuqi1129 Can you help release this Docker image to hub?

OK